lib/fluent/plugin/out_influxdb.rb

```diff
@@ -51,7 +54,8 @@ def write(chunk)
         point = {}
         point[:timestamp] = record.delete('time') || time
         point[:series] = tag
-        point[:values] = record
+        point[:tags] = record.select{|k,v| @tag_keys.include?(k)}
+        point[:values] = record.select{|k,v| !@tag_keys.include?(k)}
```

Comment:
Calling `select` is hight cost when tag_keys is empty. Following like code is better.

```rb
if tag_keys.empty?
  point[:values] = record
else
  point[:tags] = record.select{|k,v| @tag_keys.include?(k)}
  point[:values] = record.select{|k,v| !@tag_keys.include?(k)}
end
```